pkg/internal/cyberark/identity/identity.go

```diff
@@ -156,9 +156,9 @@ type advanceAuthenticationRequestBody struct {
 	// get the Identity API URL, but we set it anyway to be explicit.
 	TenantID string `json:"TenantId"`
 
-	// PersistantLogin is documented to "[indicate] whether the session should persist after the user
+	// PersistentLogin is documented to "[indicate] whether the session should persist after the user
 	// closes the browser"; for service-to-service auth which we're trying to do, we set this to true.
-	PersistantLogin bool `json:"PersistantLogin"`
+	PersistentLogin bool `json:"PersistentLogin"`
```

Comment:
This did not match the API, and thus is an error in the client: https://docs.cyberark.com/identity/latest/en/content/developer/authentication/on-demand-challenges.htm?Highlight=PersistentLogin